java/buildconf/ivy/ivy-suse.xml

```diff
@@ -92,6 +92,7 @@
         <dependency org="suse" name="xerces-j2" rev="2.11.0" />
         <dependency org="suse" name="xmlsec" rev="2.0.7" />
         <dependency org="suse" name="velocity" rev="1.5" />
+        <dependency org="suse" name="commons-lang" rev="2.6" />
```

Comment:
uhhh, we tried so hard to get rid of commons-lang 2 . Is there no posibility to use v3?

Reply:
we'll need to investigate upgrading velocity then.

Reply:
Commented out that line